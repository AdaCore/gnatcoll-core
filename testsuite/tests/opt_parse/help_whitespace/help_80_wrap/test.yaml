driver: build_run_diff
description: Check that a long help message wraps at the default 80 character limit