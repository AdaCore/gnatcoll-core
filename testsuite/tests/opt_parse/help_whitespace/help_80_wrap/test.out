Help text wrapped at 80 characters as expected